lib/Tracy/DIPanel/DIPanel.php

```diff
@@ -0,0 +1,150 @@
+      style="height: 15px"
+    />';
+    return $img . '<span class="tracy-label" >' . count($this->free_services) . ' free services + ' . count($this->premium_services) . ' premium services</span>';
```

Comment:
![Screen Shot 2019-09-03 at 1 34 12 PM](https://user-images.githubusercontent.com/1082140/64170090-8de76080-ce4f-11e9-94c0-270aa155ec2b.png)
I would change the label in the panel to: `55 + 12 services`. The current label makes the panel too wide.